### PR TITLE
Fixes to api alert history table in admin portal

### PR DESCRIPTION
**Pull request by chamilaadhi**
This PR 
- removes Abnormal API Usage alert
- updates unseen ip alert history api request
- fixes api availability issue for tenant

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin/site/themes/wso2/templates/alert-table/js/alert-table.js [hunk 1]
```diff
@@ -46,14 +46,12 @@ $( document ).ready(function() {
         } else if(selectedOptionKey== 4){
             tableName = "ORG_WSO2_ANALYTICS_APIM_ABNORMALRESPONSETIMEALERTSTREAM";
```
Comment (fazlan-nazeem) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin/site/themes/wso2/templates/alert-table/js/alert-table.js]: Are these table names correct? I checked the STATS_DB tables and it does not have ORG_WSO2_ANALYTICS_APIM prefix

Comment (chamilaadhi) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin/site/themes/wso2/templates/alert-table/js/alert-table.js]: Actually, these tables were old one. all the mappings to the latest tables are done in the /modules/stats/stats.jag 

https://github.com/wso2/carbon-apimgt/blob/6.x/features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin/modules/stats/stats.jag#L218

Comment (chamilaadhi) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin/site/themes/wso2/templates/alert-table/js/alert-table.js]: Better to remove old table names. will update the PR with repalced one

Comment (fazlan-nazeem) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin/site/themes/wso2/templates/alert-table/js/alert-table.js]: Yes. That would be better

Comment (chamilaadhi) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin/site/themes/wso2/templates/alert-table/js/alert-table.js]: updated the table names

**Pull request merged**